ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -170,6 +170,28 @@ objects:
         cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_openstack_rsa || true
         export TEST_PROVIDER='{"type":"openstack"}'
 
+        # set up openstack env vars for testing
+        function get_clouds_param() {
+          awk '/^\s+'$1':/ { print $2 }' /tmp/cluster/clouds.yaml | sed -e 's/^"//' -e 's/"$//'
+        }
+
+        if set -o | grep xtrace.*on &>/dev/null; then
+          XTRACE_ENABLED=1
+        fi
+        set +x #make extra sure we aren't echo'ing these to stdout
+        export OS_AUTH_URL="$(get_clouds_param 'auth_url')"
+        export OS_PROJECT_ID="$(get_clouds_param 'project_id')"
+        export OS_PROJECT_NAME="$(get_clouds_param 'project_name')"
+        export OS_USERNAME="$(get_clouds_param 'username')"
+        export OS_PASSWORD="$(get_clouds_param 'password')"
+        export OS_REGION_NAME="$(get_clouds_param 'region_name')"
+        export OS_ENDPOINT_TYPE="$(get_clouds_param 'interface')"
+        export OS_IDENTITY_API_VERSION="$(get_clouds_param 'identity_api_version')"
+        export OS_USER_DOMAIN_NAME="$(get_clouds_param 'user_domain_name')"
+        if [[ "$XTRACE_ENABLED" == 1 ]]; then
```

Comment:
pj-rehearse job complains about `/bin/bash: line 58: XTRACE_ENABLED: unbound variable`

Reply:
Got it set in the latest push